### PR TITLE
feat(i18n): migrate user page

### PR DESCRIPTION
**Pull request by Jasmine-liang**
Migrate User Page including:
- Share and `...` Dropdown button
- RSS feed modal
- Circle Widget
- Article/Respinse Tab
- Created xxx  articles In total xxx words

### CODE REVIEW

File: src/components/Dialogs/ShareDialog/Copy.tsx [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
-  const { lang } = useContext(LanguageContext)
-
+  const intl = useIntl()
   return (
     <section className="copy">
       <CopyToClipboard text={link}>
-        <button aria-label={translate({ id: 'copy', lang })}>
+        <button
+          aria-label={intl.formatMessage({
+            defaultMessage: 'Copy Link',
```
Comment (robertu7) [inline src/components/Dialogs/ShareDialog/Copy.tsx:14]: Leave the `description` blank to reuse as https://github.com/thematters/matters-web/pull/3252/files#diff-3e9f5ac089304045f9b77c31ea8802a522caf95a8e8f7e0942ed69971f22e569R124-R125?

File: src/components/Dialogs/ShareDialog/Copy.tsx [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
+        >
           <TextIcon icon={<IconLink16 color="grey" />} spacing="base">
             <div className="text">
               <span>
-                <Translate
-                  zh_hant="複製鏈接"
-                  zh_hans="复制链接"
-                  en="Copy Link"
+                <FormattedMessage
+                  defaultMessage="Copy Link"
```
Comment (robertu7) [inline src/components/Dialogs/ShareDialog/Copy.tsx:22]: Same comment as above

File: src/views/User/Articles/UserArticles.tsx [hunk 3]
```diff
@@ -40,16 +40,21 @@ const ArticleSummaryInfo = ({
...
   return (
     <div className="info">
-      <Translate zh_hant="創作了" zh_hans="创作了" en="Created" />
+      <FormattedMessage
+        defaultMessage="Created"
+        description="src/views/User/Articles/UserArticles.tsx"
+      />
       <span className="num">&nbsp;{articles}&nbsp;</span>
-      <Translate zh_hant="篇作品" zh_hans="篇作品" en="articles" />
+      <FormattedMessage defaultMessage="articles" description="" />
```
Comment (robertu7) [inline src/views/User/Articles/UserArticles.tsx:48]: these words (篇作品) seem to be used once, so not intended to be reused?

File: src/views/User/Articles/UserArticles.tsx [hunk 4]
```diff
@@ -40,16 +40,21 @@ const ArticleSummaryInfo = ({
...
 
-      <Translate zh_hant="累積創作" zh_hans="累积创作" en="In total" />
+      <FormattedMessage
+        defaultMessage="In total"
+        description="src/views/User/Articles/UserArticles.tsx"
+      />
       <span className="num">&nbsp;{words}&nbsp;</span>
-      <Translate zh_hant="字" zh_hans="字" en="words" />
-
+      <FormattedMessage defaultMessage="words" description="" />
```
Comment (robertu7) [inline src/views/User/Articles/UserArticles.tsx:57]: same as above about reusability, and much relying on context.

**Review by robertu7 — APPROVED** [review]

File: src/views/User/Articles/UserArticles.tsx (hunk 3 above)
Comment (Jasmine-liang) [reply src/views/User/Articles/UserArticles.tsx:48]: Legit

**Pull request merged**